src/services/ServiceLocator.ts

```diff
@@ -23,10 +23,14 @@ import { IService } from './IService';
 export class ServiceLocator {
     private services: Map<string,IService> = new Map<string, IService>();
 
-    public static Current: ServiceLocator;
+    private static _Current: ServiceLocator;
```

Comment:
```suggestion
    private static current: ServiceLocator;
```
no need to be with underscore, this is typescript